### PR TITLE
Changed_OEL_to_OL

### PR DESCRIPTION
**Pull request by ManjushaVaidya**
## What Changed?

- Changed references of OEL to OL

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**
- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

**Review by abhilasha-narendra — COMMENTED** [review]
Looks good.

**Comment by ManjushaVaidya** [comment]
> I think we can remove the 6.x platform-related note from all the DBInfra products, as we maintain only the latest version for all these products.

We have listed all the versions in the supported platforms table (https://www.enterprisedb.com/docs/pgbouncer/latest/01_installation/). Earlier Centos 6 was also listed, but after this comment removed Centos 6 from table and added this note. This applies to all DB-Infra products

File: product_docs/docs/postgis/1.0/01_installing_postgis.mdx [hunk 1]
```diff
@@ -25,8 +25,8 @@ The following table lists the latest PostGIS versions and their corresponding Ad
 | `PostGIS 2.3`       | Advanced Server 9.6 and 10             | RHEL/CentOS 7 - x86_64 <br/>RHEL/CentOS 7 - ppc64le <br/>Windows 64 x86 Interactive Installer                                                                                                                              |
 
 !!! Note
-    PostGIS is not supported on CentOS/RHEL/OL 6.x platforms. EDB strongly recommends migrating all EDB products running on these platforms to a supported platform.
-
+    PostGIS is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended that EDB products running on these platforms be migrated to a supported platform.
```
Comment (drothery-edb) [inline product_docs/docs/postgis/1.0/01_installing_postgis.mdx:28]: I would not change the second sentence. Keep it in active voice.

File: product_docs/docs/slony/1.0/01_installation/index.mdx [hunk 2]
```diff
@@ -22,14 +22,10 @@ The following table lists the latest Slony Replication versions and their corres
 | `Slony Replication 2.2`                 | EDB Postgres Advanced and PostgreSQL Server 10 and 9.6 | RHEL 7 - x86_64 <br/>RHEL 7 - ppc64le <br/>Windows x86_64 and Linux graphical installer                                                                                                     |
 
 !!! Note
-    Slony is not supported on CentOS/RHEL/OL 6.x platforms. EDB strongly recommends migrating all EDB products running on these platforms to a supported platform.
+    Slony is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended to migrate all EDB products running on these platforms to a supported platform.
```
Comment (drothery-edb) [inline product_docs/docs/slony/1.0/01_installation/index.mdx:25]: Don't change second sentence

**Review by drothery-edb — COMMENTED** [review]
I left a couple comments

File: product_docs/docs/slony/1.0/01_installation/index.mdx (hunk 2 above)
Comment (ManjushaVaidya) [reply product_docs/docs/slony/1.0/01_installation/index.mdx:25]: I have changed to maintain consistency as it is "Slony is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended to migrate all EDB products running on these platforms to a supported platform" in all guides.

File: product_docs/docs/postgis/1.0/01_installing_postgis.mdx (hunk 1 above)
Comment (ManjushaVaidya) [reply product_docs/docs/postgis/1.0/01_installing_postgis.mdx:28]: I have changed to maintain consistency as it is "Slony is no longer supported on CentOS/RHEL/OL 6.x platforms. It is strongly recommended to migrate all EDB products running on these platforms to a supported platform" in all guides.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**